db/iscn.go

```diff
@@ -34,10 +38,13 @@ func QueryISCN(pool *pgxpool.Pool, iscn ISCN, page PageRequest) (ISCNResponse, e
 	ctx, cancel := GetTimeoutContext()
 	defer cancel()
 
-	rows, err := conn.Query(ctx, sql, iscn.Iscn, iscn.Owner, iscn.Keywords, iscn.Fingerprints, iscn.Stakeholders,
-		page.After(), page.Before(), page.Limit)
+	rows, err := conn.Query(
+		ctx, sql,
+		query.IscnID, query.Owner, query.Keywords, query.Fingerprints, query.StakeholderID, query.StakeholderID,
```

Comment:
The second one should be StakeholderName?
```suggestion
		query.IscnID, query.Owner, query.Keywords, query.Fingerprints, query.StakeholderID, query.StakeholderName,
```

Reply:
Is that normal for a query by keyword, owner, or stakeholder takes about 5s to complete?

Reply:
No, it should take not more than 2s.